client/src/infographic/bubble_definitions.js

```diff
@@ -2,14 +2,15 @@ import _ from "lodash";
 
 import { create_text_maker } from "../models/text.js";
 
+import svgs from "./bubble_svgs.js";
+
 import { infograph_href_template } from "./infographic_link.js";
 
 import text from "./bubble_definitions.yaml";
-import svgs from "./bubble_svgs.yaml"; // TODO: use icon componenets instead (some of these are already icon components, some not)
 
 const text_maker = create_text_maker(text);
 
-const get_svg_content = (bubble_id) => _.get(svgs, `${bubble_id}.text`);
+const get_svg_content = (bubble_id) => _.get(svgs, bubble_id);
```

Comment:
Not necessary anymore, cleaner I think to drop this helper func  and `bubble_svgs.js`, just import the icons and put them directly in to `base_configs`.